src/main/java/net/snowflake/client/core/SessionUtil.java

```diff
@@ -998,6 +1064,14 @@ public static void deleteMfaTokenCache(String host, String user) {
     CredentialManager.getInstance().deleteMfaTokenCache(host, user);
   }
 
+  private static void deleteOAuthAccessTokenCache(String host, String user) {
+    CredentialManager.getInstance().deleteOAuthAccessTokenCache(host, user);
```

Comment:
cannot we expose static methods directly from CredentialManager? under the hood it's instance could be used and we don't need to provide such helper methods?

Reply:
Done.